packages/core/lib/debug/printer.js

```diff
@@ -86,6 +87,12 @@ class DebugPrinter {
     );
   }
 
+  warnIfNoSteps() {
+    if (this.session.view(trace.steps).length === 0) {
+      this.config.logger.log("Warning: This transaction has no trace steps.");
```

Comment:
Can we start standardizing debugger messaging? Like making `Warning:` bold and punctuating with blank lines above/below? Does it makes sense to make this look like `--stacktrace-extra` warning?

If possible, it might also be a good opportunity to add some helpful info here... like, what could cause the lack of trace steps, and is there anything the user can do to fix that?

Reply:
> Can we start standardizing debugger messaging? Like making Warning: bold and punctuating with blank lines above/below? Does it makes sense to make this look like --stacktrace-extra warning?

Hm, we don't do that with our other warning (some contracts couldn't be identified).  Should we?  Bold sounds like a good idea, I dunno about additional blank lines (I think it's OK for both warnings to appear together?).

> If possible, it might also be a good opportunity to add some helpful info here... like, what could cause the lack of trace steps, and is there anything the user can do to fix that?

I mean, the two reasons I can think of are:
1. There are legitimately no trace steps (you decided to inspect a transfer between two externally-owned accounts for some perverse reason)
2. Your ethereum client failed to return trace steps for some reason!

I'm not really sure what more there is to say about either of these?

Reply:
Hm, well, we could say something like that. Here's a rough take:

```
Warning: this transaction has no trace steps. This may happen if you are attempting to debug a transaction between two externally-owned accounts, or if the node you are connecting to failed to produce a trace for some reason. Please check your configuration and try again.
```

Reply:
OK.  Should warnings be bolded too?

Reply:
Yep! The goal is to get users to read them :)